en/developer/frameworks/articles/search/03-queries-and-filters/02-building-search-queries-and-filters.markdown

```diff
@@ -25,14 +39,14 @@ To satisfy the dependencies of the example code presented here, get references t
 - `com.liferay.portal.search.query.Queries`
 
 ```java
-@Reference
-protected Queries _queries;
+	@Reference
```

Comment:
Because we're not in a bullet or numbered list, there's no need to indent.

Also, should we default to using private fields for these?

Reply:
No. It's passing SF as-is.

Reply:
> Because we're not in a bullet or numbered list, there's no need to indent.

OK.